app/Support/Cache/CacheKey.php

```diff
@@ -49,6 +54,34 @@ public static function buildUserRecentGamesCacheKey(string $username): string
         return self::buildNormalizedUserCacheKey($username, "recent-games");
     }
 
+    /**
+     * Constructs a normalized cache key.
+     *
+     * This function creates a cache key for a specific entity with a given kind and optional parameters.
+     * Cache keys follow the format: "entityKind:12345:keyKind:{param1}:{param2}".
+     *
+     * @param string     $entityKind the kind of entity to cache data for (eg: "user", "game", "achievement")
+     * @param string|int $identifier the unique id associated with this entity (eg: 12345, "Scott")
+     * @param string     $keyKind the specific kind of data being cached (eg: "card-data")
+     * @param array      $params optional parameters for the cache key
+     *
+     * @return string the constructed cache key
```

Comment:
I tend to omit `@param` and `@return` in docblock if the description doesn't add context for highly complex structures. Usually it's easy to determine what parameters are for and/or what methods will return; especially since PHP allows us to specify types. It'S easier to maintain, too. We removed most of them when we introduced types.

However, if you think it's still worth keeping those I'd prefer the non-aligned variants as that's how the other docblocks formatted.

Reply:
That makes sense. Some of these are definitely redundant, I didn't know if it was a good practice to only keep around the ones that add useful context while omitting the ones that are redundant. I've trimmed out a lot of these in https://github.com/RetroAchievements/RAWeb/pull/1700/commits/503bff6530c5ea54824c288909886aa2d3df93ac so hopefully the only params that could be ambiguous are still documented.